Package.swift

```diff
@@ -306,7 +322,10 @@ let package = Package(
         .library(
             name: "AWSPinpointAnalyticsPlugin",
             targets: ["AWSPinpointAnalyticsPlugin"]
-        )
+        ),
+        .library(
+            name: "AmplifyAsyncTesting",
+            targets: ["AmplifyAsyncTesting"]),
```

Comment:
With this change and removing the AsyncExpectation from the HostApps will break builds for the integration tests. The follow up to this is to add AmplifyAsyncTesting to the HostApp and also XCTest.framework with "Do not Embed" (fails to compile without this).
<img width="1169" alt="image" src="https://user-images.githubusercontent.com/1365977/187762826-1f9139c0-9dd6-4f76-8ab9-7009d8cb3816.png">

Reply:
Just adding AmplifyAsyncTesting shows this error
<img width="517" alt="image" src="https://user-images.githubusercontent.com/1365977/187764338-10923a32-165a-4647-954d-0e72950ad857.png">
Adding XCTest.framework shows duplicate error
Set to "Do not Embed" compiles file

Reply:
yes, normally XCTest is not added to an app project. this could also be added to linking but this works too.